src/jit/emit.h

```diff
@@ -638,7 +638,7 @@ class emitter
 
     private:
 #if defined(_TARGET_XARCH_)
-        unsigned _idCodeSize : 4; // size of instruction in bytes
+        unsigned _idCodeSize : 4; // size of instruction in bytes, max is 15 bytes, because of the Intel decoder limit.
```

Comment:
This comment is a little ambiguous to me. I don't know what "the Intel decoder limit" is. Is it the maximum size a single Intel x86/x64/xarch instruction can be? Or a limit in our decoder for Intel xarch instructions?

Reply:
It is Intel processors decoders, so if a processor sees an instruction longer it will throw "invalid opcode" (it would be "legal" because Intel doesn't forbid repeated prefixes, but won't be decodable and executable).
[from Intel® 64 and IA-32 Architectures Software Developer’s Manual Volume 2
](https://www.intel.com/content/dam/www/public/us/en/documents/manuals/64-ia-32-architectures-software-developer-instruction-set-reference-manual-325383.pdf):
2.3.11 AVX Instruction Length
The AVX instructions described in this document (including VEX and ignoring other prefixes) do not exceed 11 bytes in length, but may increase in the future. The maximum length of an Intel 64 and IA-32 instruction remains 15 bytes.

Could you help me to rephrase the comment please?

Reply:
how about:
```
// size of instruction in bytes. Max size of an Intel instruction is 15 bytes.
```

Reply:
Looks good, fixed. Thank you.